docs/production-deployment/cloud/api-keys.mdx

```diff
@@ -87,15 +82,24 @@ You must enable API Key access to allow the creation of API Keys.
 
 :::
 
-Global Administrators can monitor, manage, update access, and delete API Keys for any user within their account.
+Global Administrators can monitor, manage, disable, and delete API Keys for any User or Service Account within their account.
 
 To manage your accounts API Keys:
 
 1. [Login](https://cloud.temporal.io/) to the Cloud UI.
 2. [Select **Settings** and choose **API Keys**](https://cloud.temporal.io/settings/api-keys).
 
-From here, you can update access to the API Key for an account using the **Enable** and **Disable** toggle.
-You can also disable or delete an individual User API Key using the vertical ellipsis at the right of the API Keys row.
+You can also disable or delete an individual API Key using the vertical ellipsis at the right of the API Keys row. Disabling API Keys disables the creation of API Keys. Existing API Keys can be used to authenticate into Temporal Cloud until they are either disabled, deleted or expire.
```

Comment:
Feels second part of this row "Disabling API Keys..." should be on its own line

Reply:
ack - making this change